app.go

```diff
@@ -1000,3 +1053,76 @@ func (app *App) Fields() (map[string]*FieldInfo, error) {
 	}
 	return ret, nil
 }
+
+func (app *App) createCursor(fields []string, query string, size uint64) ([]byte, error) {
+	type cursor struct {
+		App    uint64   `json:"app"`
+		Fields []string `json:"fields"`
+		Size   uint64   `json:"size"`
+		Query  string   `json:"query"`
+	}
+	data := cursor{App: app.AppId, Fields: fields, Size: size, Query: query}
+	jsonData, _ := json.Marshal(data)
+	url := app.createUrl("records/cursor", "")
+	request, err := app.NewRequest("POST", url.String(), bytes.NewBuffer(jsonData))
+	if err != nil {
+		return nil, err
+	}
+	response, err := app.do(request)
+
```

Comment:
Could you please remove this blank line, and using blank line effectively and sensibly. And also do this with other functions.